sdk/core/core-lro/package.json

```diff
@@ -69,11 +69,11 @@
     "build": "npm run clean && dev-tool run build-package && dev-tool run extract-api",
     "build:samples": "echo Obsolete",
     "build:test": "echo skipped. actual commands inlined in browser test scripts",
-    "check-format": "dev-tool run vendored prettier --list-different --config ../../../.prettierrc.json --ignore-path ../../../.prettierignore \"src/**/*.{ts,cts,mts}\" \"test/**/*.{ts,cts,mts}\" \"*.{js,cjs,mjs,json}\"",
```

Comment:
nit: Do we not need to specify `cts`, `mts`, and `ts`? Today I dont believe we have this in core-lro but other core libraries do and one day we may add the same kinds here

This is usually done to fix any dual-package hazards https://github.com/isaacs/tshy?tab=readme-ov-file#dual-package-hazards or to ship CJS / ESM specific code